### ISSUE TITLE
Updating Steam Deck to 3.4.2 stops Mod Organizer from launching games

### ISSUE THREAD

**Issue by rbroab** [rockerbacon/modorganizer2-linux-installer#431]
After updating Steam Deck to 3.4.2, games will no longer launch. 

Mod Organizer 2 will launch normally from game mode or desktop mode. However, once MO2 is open, trying to launch a game will actually try and open Mod Organizer 2 again. Trying to launch NVSE, New Vegas exe, Fallout New Vegas Launcher all have the same issue. A message will open- “An instance of Mod Organizer is already running.” Closing this message brings you back to the original instance of MO2.

Running other programs through MO2 seem to work, such as FNVEdit work without issue.

**Comment by fromthedarkestpast** [comment]
Same problem, with New Vegas. 
Tried reinstalling xNVSE - 4gb patch - MO2, remained the same, MO2 launcher tries to open itself.
Wondering if I should try it on other games until any fixes.

**Comment by rbroab** [comment]
Oblivion launches normally. Haven’t tested any other Fallout or Elder Scrolls. 

Could be a New Vegas only issue

**Comment by mjboring** [comment]
Same issue after updating Steam OS to 3.4.1 and 3.4.2.

**Comment by BombFromMars** [comment]
Also have this issue

**Comment by ImCrumby** [comment]
Having the same issue on 3.4.4. Fails to launch NVSE or TTW. Launched xLODGen and FNVEdit without problems however.

**Comment by ImCrumby** [comment]
I wrote a guide to fix the problem https://docs.google.com/document/d/1NhLgjyP8UmbW8HttsqPzPThvPQtpydLcrF00ZY5tXYQ/edit?usp=sharing

**Comment by Martan404** [comment]
Skyrim has worked for me with no problems

**Comment by faultybat** [comment]
Hi there I think I have found one possible reason why Fallout NV is failing to launch post-update. 
After reinstalling Mod organizer and getting fallout NV to launch again, I looked at the differences between the falloutprefs.ini files between the old install and the new installation and noticed that the sD3DDevice variable had been changed.  

In the old installed  sD3DDevice was called "AMD RADV VANGOGH" whereas in the new installed the sD3DDevice is called "AMD Custom GPU 0405 (RADV VANGOGH)"`. 

As I have reinstalled Mod organizer there might be other reasons why it is now fixed, but I suggest that anyone who is still experiencing this problem check the falloutprefs.ini file in the Mod organizer 2 profiles, and the sD3DDdevice to the updated name. 

`sD3DDevice="AMD Custom GPU 0405 (RADV VANGOGH)"`
